test/generate_tests.py

```diff
@@ -156,3 +156,69 @@ def test_main_drizzle(self):
 			os.remove(test_cosmo_folder+'npy_files/img_%d.npy'%(i))
 		os.rmdir(test_cosmo_folder+'npy_files')
 		os.rmdir(output_folder)
+
+	def test_h5(self):
+		# Test that the h5 kwarg generates the required h5 files
+		old_sys = copy.deepcopy(sys.argv)
+		n_generate = 21
```

Comment:
I think you're currently not testing the 100-image batching logic; maybe try generating 101 instead of 21 images? The tests will take a bit longer, but we can always optimize the test config to be a simpler task later.